doc/api/child_process.markdown

```diff
@@ -758,13 +511,368 @@ throw.  The [`Error`][] object will contain the entire result from
   * `signal` {String} The signal used to kill the child process
   * `error` {Error} The error object if the child process failed or timed out
 
-`spawnSync` will not return until the child process has fully closed. When a
-timeout has been encountered and `killSignal` is sent, the method won't return
-until the process has completely exited. That is to say, if the process handles
-the `SIGTERM` signal and doesn't exit, your process will wait until the child
+The `child_process.spawnSync()` method is generally identical to
+`child_process.spawn()` with the exception that the function will not return
+until the child process has fully closed. When a timeout has been encountered
+and `killSignal` is sent, the method won't return until the process has
+completely exited. Note that if the process intercepts and handles the
+`SIGTERM` signal and doesn't exit, the parent process will wait until the child
 process has exited.
 
+## Class: ChildProcess
+
+Instances of the `ChildProcess` class represent spawned child processes.
+
+`ChildProcess` is an [`EventEmitter`][].
+
+Child processes always have three associated streams: `child.stdin`,
+`child.stdout`, and `child.stderr`.  These might be shared with the stdio
+streams of the parent process, or they may be separate stream objects
+which can be piped to and from.
+
+Instances of `ChildProcess` are not intended to be created directly. Rather,
+Use the [`child_process.spawn()`][], [`child_process.exec()`][],
+[`child_process.execFile()`][], or [`child_process.fork()`][] methods (or
+their synchronous counterparts) to create instances of `ChildProcess`.
```

Comment:
hmm.. good point. It looks like the return value of the sync variants isn't documented at all currently :-/